src/app/migration/src/src/app/services/pagination/pagination.service.ts

```diff
@@ -0,0 +1,43 @@
+import * as _ from 'lodash';
+
+export class PagerService {
```

Comment:
Change this to pagination, keep consistent

Reply:
done